common/src/main/java/io/netty/util/ReferenceCountUtil.java

```diff
@@ -48,6 +49,7 @@
      */
     @SuppressWarnings("unchecked")
     public static <T> T retain(T msg, int increment) {
+        ObjectUtil.checkPositive(increment, "Increment");
```

Comment:
Use lowercase for the name as it should match the variable name. This is true for all usages

Reply:
okay

Reply:
Done